devops/systemd.conf

```diff
@@ -14,7 +14,7 @@ Environment=DD_GIT_COMMIT_SHA="{{ lookup('env', 'GIT_SHA') }}"
 Environment=DD_GIT_REPOSITORY_URL="github.com/transitmatters/gobble"
 Environment=DD_PROFILING_ENABLED="true"
 Environment=DD_LOGS_INJECTION="true"
-ExecStart=/home/ubuntu/.local/bin/poetry run ddtrace-run python3 src/gobble.py
+ExecStart=/home/ubuntu/.local/bin/uv run ddtrace-run src/gobble.py
```

Comment:
I think the python3 may still be needed here

Reply:
Ack - assume the same should be changed back for the playbook?